packages/editor-ui/src/components/Node.vue

```diff
@@ -1405,6 +1433,13 @@ export default defineComponent({
 		margin-top: calc(var(--spacing-l) * -1);
 		margin-left: 0;
 	}
+
+	&[data-endpoint-node-type='n8n-nodes-base.switch'] {
+		max-width: calc(var(--stalk-size) - (var(--endpoint-size-small)));
```

Comment:
Why does this styling only apply for switch nodes? + why is `--stalk-size` different?

Reply:
Because users can now label their outputs so it's not just a number. And since it's using `small` endpoint size, the stalk-size would also get reduced hence the overwrite. I've added a comment to explain the reasoning